save-common/src/commonMain/kotlin/com/saveourtool/save/core/config/TestConfig.kt

```diff
@@ -122,12 +122,15 @@ data class TestConfig(
      * @return an update this [TestConfig]
      */
     fun processInPlace(createPluginConfigList: (TestConfig) -> List<PluginConfig>): TestConfig {
+        // need to process parent first
+        this.parentConfig?.processInPlace(createPluginConfigList)
```

Comment:
Why is this needed here? Shouldn't we start iterating from parent here: https://github.com/saveourtool/save-cli/blob/93a615e0f0c5b48e386ff86e8485e59e35d4bcb1/save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt#L71-L74
And have parent config already processed by this time?

Reply:
it's required when testRootDir is sub folder, for example:
1. `./save.toml` contains information how to run
2. run tests `save-cli ./subfolder/ ...`.
3. save-cli processes `./subfolder/save.toml`, but it doesn't contain information how to run the test (`no exec_cmd in config`)

Reply:
will add a test with example which was provided by @akuleshov7 for original issue

Reply:
Yes, but I thought that when we call `configFromFile() -> discoverConfigWithParents -> getTestConfigFromTomlFile` we are processing all parent directories of the current file, so it seems that the original intention was to process them there. Probably, there is a bug in there somewhere.

Reply:
discussed offline -- raised https://github.com/saveourtool/save-cli/issues/415 for it
probably need to fix argument descriptions